src/entities/WeeklyDebrief.ts

```diff
@@ -0,0 +1,42 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import { prop as Property, DocumentType } from '@typegoose/typegoose';
+import mongoose from 'mongoose';
+import { Article } from './Article';
+
+@ObjectType({ description: 'The Weekly Debrief Model' })
+export default class WeeklyDebrief {
+  @Field(() => ID)
+  id: string;
+
+  @Field()
+  @Property()
+  uuid: string; // uuid of user associated with this WeeklyDebrief
```

Comment:
Does this show up as a comment in the Apollo GraphQL interface? If not, try to make it comment that does, it may be a param in the decorator, not sure though

Reply:
+1

Reply:
Is there a way to do this using TypeGraphQL? I've been looking around in their docs but I'm not finding anything.